modules/social_features/social_group/modules/social_group_welcome_message/social_group_welcome_message.services.yml

```diff
@@ -0,0 +1,8 @@
+services:
+
+  # Defines service for overriding configs
+  social_group_welcome_message.overrider:
+    class: Drupal\social_group_welcome_message\SocialGroupWelcomeMessageConfigOverride
+    arguments: ['@config.factory', '@module_handler']
```

Comment:
Add the **string_translation** service if it is possible.

Reply:
Added